common/pkg/cgroups/cpuset_linux.go

```diff
@@ -18,37 +15,6 @@ func getCpusetHandler() *linuxCpusetHandler {
 	return &linuxCpusetHandler{}
 }
 
-// Apply set the specified constraints.
-func (c *linuxCpusetHandler) Apply(ctr *CgroupControl, res *cgroups.Resources) error {
-	if ctr.cgroup2 {
-		man, err := fs2.NewManager(ctr.config, filepath.Join(cgroupRoot, ctr.config.Path))
-		if err != nil {
-			return err
-		}
-		return man.Set(res)
-	}
-	path := filepath.Join(cgroupRoot, CPUset, ctr.config.Path)
-	return c.CPUSet.Set(path, res)
-}
-
-// Create the cgroup.
-func (c *linuxCpusetHandler) Create(ctr *CgroupControl) (bool, error) {
```

Comment:
OK, removing this is one way to deal with https://github.com/containers/container-libs/pull/417#discussion_r2482821183 .

… but, please, make a permanent record of the rationale. Why is removing the `…CopyFromParent` code fine? The commit message doesn’t say. (I don’t understand cgroups and I have no opinion. It’s quite possible that the manual copies were a bug.)

If this should be obvious to anyone minimally familiar with cgroups, I’m happy to defer to a different reviewer.